src/code42cli/cmds/auditlogs.py

```diff
@@ -155,11 +201,39 @@ def send_to(
     )
 
 
-def _search(sdk, format, **filter_args):
-
+def _search(sdk, format, save_checkpoint, **filter_args):
     formatter = OutputFormatter(format, AUDIT_LOGS_DEFAULT_HEADER)
     response_gen = sdk.auditlogs.get_all(**filter_args)
+    events = _get_all_audit_log_events(response_gen)
+    event_count = len(events)
+    if not event_count:
+        click.echo("No results found.")
+        return
+    elif event_count > 10:
+        click.echo_via_pager(formatter.get_formatted_output(events))
+    else:
+        formatter.echo_formatted_list(events)
+    if save_checkpoint:
+        ts = _parse_audit_log_timestamp_string_to_timestamp(events[0]["timestamp"])
```

Comment:
So we are still getting the latest timestamp?  Is this a problem? I was expecting to see logic like this imported here: https://github.com/code42/c42eventextractor/blob/master/src/c42eventextractor/extractors.py#L169